### PR TITLE
ThermoRawFileConverter tool: always returns collections

### PR DESCRIPTION
**Pull request by CarlosHorro**
The tool always produces now collections in order to avoid problems working with workflows and variable inputs.
Changed some input and output names in order to have clearer names.
Some code cleaning.

### CODE REVIEW

**Comment by CarlosHorro** [comment]
Hi,

I've tested the tool with `planemo test ` in my computer with the last galaxy version and python 3.7, and all tests pass successfully. How can I check the details of the failed ` Combine chunked tests results - Check status of combined status` task in order to know what is going wrong in the automatic test?

Greetings,
Carlos

File: tools/ThermoRawFileParser/thermo_converter.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<tool id="thermo_raw_file_converter" name="Thermo" version="1.2.3">
+<tool id="thermo_raw_file_converter" name="Thermo" version="1.2.3.5">
```
Comment (bgruening) [inline tools/ThermoRawFileParser/thermo_converter.xml:1]: can we change this to `1.2.3+galaxy0`.  The suffix is somehow our conda-build string.

**Comment by bgruening** [comment]
Other than the version number, this looks good to me. Thanks @CarlosHorro!

**Comment by bgruening** [comment]
Merci!

**Pull request merged**